style.css

```diff
@@ -32,15 +33,16 @@ header {
 
 .menu li {
     list-style-type: none;
-    padding: 0.5%;
+    padding: 11.5%;
```

Comment:
In general I would avoid using % for padding and margin. It means you can't control exactly how big the spacing will be, which will give inconsistent gaps between elements. Usually, designers will have fixed pixel sizes that they use for all spacing to keep it consistent.